jcl/src/java.base/share/classes/java/lang/ref/Reference.java

```diff
@@ -235,4 +235,17 @@ public static void reachabilityFence(java.lang.Object ref) {
 }
 /*[ENDIF]*/
 
+/*[IF Java11]*/
+/**
+ * This method will always throw CloneNotSupportedException. A clone of this instance will not be returned 
+ * since a Reference cannot be cloned. Workaround is to create a new Reference.
+ * 
+ * @throws CloneNotSupportedException
+ *
+ * @since 11
+ */
+protected Object clone() throws CloneNotSupportedException {
+	throw new CloneNotSupportedException(com.ibm.oti.util.Msg.getString("K0900")); //$NON-NLS-1$
```

Comment:
The use of K0900 needs to be preceded by a java preprocessor command to declare the external message, otherwise the message won't be defined. The only purpose adding it to ExternalMessages-MasterIndex.properties is so we have a single place that lists all the keys in use.

Reply:
fixed.